mkt/fireplace/tests/test_views.py

```diff
@@ -168,13 +169,15 @@ def setUp(self):
         self.url = reverse('fireplace-multi-search-api')
         self.website = website_factory()
         self.website.popularity.add(WebsitePopularity(region=0, value=666))
+        self.website.save()
         self.webapp = Webapp.objects.get(pk=337141)
+        self.webapp.save()
         self.extension = Extension.objects.create(name='test-ext-lol')
         self.extension.versions.create(status=STATUS_PUBLIC)
         self.extension.popularity.add(ExtensionPopularity(region=0, value=999))
-        self.reindex(Extension)
-        self.reindex(Webapp)
-        self.reindex(Website)
+        self.extension.save()
```

Comment:
Why this change ? `reindex()` would call `save()` + `refresh()` I think, isn't that what we want here ?